### DISCUSSION TITLE
tr/datasets/detect/african-wildlife/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#17944] [Docs]
# tr/datasets/detect/african-wildlife/

Bilgisayarla görme modellerini eğitmek için bufalo, fil, gergedan ve zebra görüntüleri içeren Afrika Vahşi Yaşam Veri Setimizi keşfedin. Araştırma ve koruma için idealdir.

https://docs.ultralytics.com/tr/datasets/detect/african-wildlife/

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for sharing your interest in Ultralytics 🚀! Your input sounds very exciting, and we appreciate you highlighting resources.

If this is a ❓ Question about datasets, model training, or any specific functionality, please provide further details about your use case to ensure we can guide you more effectively. If you are developing or using custom datasets like the African Wildlife Dataset mentioned, ensure that everything aligns with our [custom dataset setup documentation](https://docs.ultralytics.com/datasets/) for the best results.

If this is a 🐛 Bug Report, could you kindly share a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help the team quickly identify and address the issue?

For further inquiries, we encourage you to connect with the vibrant Ultralytics community:
- Join real-time discussions on [Discord](https://discord.com/invite/ultralytics) 🎧.
- Explore in-depth threads on [Discourse](https://community.ultralytics.com).
- Contribute to discussions or ask questions on our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade

To ensure you’re equipped with the latest features and fixes, upgrade to the newest `ultralytics` package version in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

For hands-on exploration of YOLO and datasets, the following environments are available:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud**. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/).
- **Amazon**. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/).
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>.

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

Rest assured that an Ultralytics engineer will review and support your discussion shortly. In the meantime, feel free to elaborate further to help address your queries or ideas more effectively. 😊
